openapi/index.yaml

```
@@ -394,6 +419,96 @@ definitions:
         format: url
     required:
       - items
+  Group:
+    type: object
+    properties:
+      built_in:
+        type: boolean
+      description:
+        type: string
+      display_name:
+        type: string
+      external_id:
+        type: string
+      group_contract_type:
+        $ref: "#/definitions/GroupType"
+      id:
+        type: string
+      name:
+        type: string
+      type:
+        type: string
+    required:
+      - display_name
+  GroupType:
+    type: string
+    x-extensible-enum:
+      - custom
+      - system
+      - external
+  Subscription:
+    type: object
+    properties:
+      allow_tracing:
+        type: boolean
+      created_date:
+        type: string
+        format: "date-time"
+      display_name:
+        type: string
+      end_date:
+        type: string
+        format: "date-time"
+      expiration_date:
+        type: string
+        format: "date-time"
+      id:
+        type: string
+      name:
+        type: string
+      notification_date:
+        type: string
+      owner_id:
+        type: string
+      primary_key:
+        type: string
+      scope:
+        type: string
+      secondary_key:
+        type: string
+      start_date:
+        type: string
+        format: "date-time"
+      state:
+        $ref: "#/definitions/SubscriptionState"
+      state_comment:
+        type: string
+      type:
+        type: string
```

```suggestion
      allow_tracing:
        type: boolean
      created_date:
        type: string
        format: "date-time"
      display_name:
        type: string
      end_date:
        type: string
        format: "date-time"
      expiration_date:
        type: string
        format: "date-time"
      id:
        type: string
      name:
        type: string
      notification_date:
        type: string
      owner_id:
        type: string
      scope:
        type: string
      start_date:
        type: string
        format: "date-time"
      state:
        $ref: "#/definitions/SubscriptionState"
      state_comment:
        type: string
      type:
        type: string
```

Same suggestion as for Groups. We need to hide the keys here, we will retrieve them by a subsequent call to
GetSubscriptionKeys(subscriptionId).